destalinator.py

```diff
@@ -155,14 +155,14 @@ def archive(self, channel_name):
             payload = self.slacker.archive(channel_name)
             if payload['ok']:
                 self.logger.debug("Slack API response to archive: %s", json.dumps(payload, indent=4))
-                self.logger.info("Archived %s", channel_name)
+                self.action("Archived {}".format(channel_name))
```

Comment:
What's the difference between `logger.info` and `action` here?

Reply:
`action` was being used by the `warner` and all it does is prepend a "[Action]" to the beginning of the log.  It still does `info` output, but I saw it as a place to make it more common to other actions.